dictionaries/pt_br.dictionary.itop.core.php

```diff
@@ -33,15 +33,15 @@
 	'Core:AttributeLinkedSet' => 'Array de objetos',
 	'Core:AttributeLinkedSet+' => 'Qualquer tipo de objetos da mesma classe ou subclasses',
 
-	'Core:AttributeLinkedSetDuplicatesFound' => 'Duplicates in the \'%1$s\' field : %2$s~~',
+	'Core:AttributeLinkedSetDuplicatesFound' => 'Duplicatas no campo \'%1$s\' : %2$s~~',
```

Comment:
```suggestion
	'Core:AttributeLinkedSetDuplicatesFound' => 'Duplicatas no campo \'%1$s\' : %2$s',
```

Please remove `~~` at the end of translated entries. The `~~` are here to help translators find the untranslated entries :)
Same comment for all other entries below, I didn't put a comment on each of them.